package.json

```diff
@@ -1,81 +1,25 @@
 {
   "name": "weather-report",
-  "version": "1.0.0",
-  "description": "Your city's weather info, one click away, is just awaiting for you!",
-  "directories": {
-    "docs": "docs"
-  },
+  "version": "0.1.0",
+  "private": true,
```

Comment:
why remove description?

Reply:
also, these ones are good to have:

![image](https://user-images.githubusercontent.com/36519478/194728037-6f614047-69d0-4160-83c4-68297f5aa2ac.png)

Please not remove